snow/engine/snowman/bootstrap/bootstrapper_test.go

```diff
@@ -83,12 +84,11 @@ func newConfig(t *testing.T) (Config, ids.NodeID, *common.SenderTest, *block.Tes
 		Sender:                         sender,
 		BootstrapTracker:               bootstrapTracker,
 		Timer:                          &common.TimerTest{},
-		AncestorsMaxContainersSent:     2000,
 		AncestorsMaxContainersReceived: 2000,
 		SharedCfg:                      &common.SharedConfig{},
 	}
 
-	snowGetHandler, err := getter.New(vm, commonConfig)
+	snowGetHandler, err := getter.New(ctx, vm, sender, time.Second, 2000)
```

Comment:
There were a number of places that we weren't specifying the maxTime previously.